integration-tests/utils/catalog-utils.js

```diff
@@ -14,6 +14,7 @@ const eitherThrow = (testFailure, { cleanupFailure, cleanupMessage }) => {
 //chance to dataset with asses
```

Comment:
any idea what this means? maybe "Change to dataset with access"?

Reply:
I have no clue. It was unrelated to my change so I didn't want to touch it.

Reply:
Not having a comment sounds preferable to having a confusing comment :)